build-system/compile/check-for-unknown-deps.js

```diff
@@ -24,12 +24,12 @@ const {red, cyan, yellow} = require('ansi-colors');
  * reference module imports. If any are found, that means Closure couldn't
  * import the module correctly.
  *
- * @return {!Stream}
+ * @return {!require('stream')}
```

Comment:
It seems weird to `require` a module only to use a type in JSDoc. I'm concerned this will add to code bloat and increased load time. What if we avoid this by just leaving the JSDoc as is and ignoring the type error?

(Same comment applies to other files too.)

Reply:
I'll remove this and just add the type to a typescript definition file.